packages/wdio-cli/src/config.js

```diff
@@ -55,14 +55,16 @@ WebdriverIO CLI runner
 
 Usage: wdio [options] [configFile]
 Usage: wdio config
-Usage: wdio repl <browserName>
+Usage: wdio repl <browserName> [appiumConfigFile]
```

Comment:
I rather prefer a Chrome example here
```suggestion
Usage: wdio repl [options] <browserName>
```